packages/ui/src/app/components/OnboardingOverlay/OnboardingOverlay.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+
+import { OnBoardingOverlay } from '@/app/components/OnboardingOverlay/OnBoardingOverlay'
+import { TemplateBlock } from '@/common/components/storybookParts/previewStyles'
+
+export default {
+  title: 'App/OnboardingOverlay',
+  component: OnBoardingOverlay,
+} as Meta
+
+const Template: Story = (args) => (
+  <TemplateBlock>
+    <OnBoardingOverlay {...args} />
+  </TemplateBlock>
+)
```

Comment:
I tried to test this on the storybook and, at least for me, it wasn't rendering anything. I would add some controls here to simulate the change in the onboarding status. This way we could test or change the UI quickly.

Reply:
Yeah, I forgot to update story after hook integration, I've take care of it and it should be working now :)